Libraries/src/Amazon.Lambda.S3Events/Amazon.Lambda.S3Events.csproj

```diff
@@ -4,9 +4,9 @@
 
```

Comment:
Can you take the `AWSSDK.S3` reference to 3.7 here too?

Reply:
Fixed